card/mod/05_standard/set/all/rich_html/content.rb

```diff
@@ -111,8 +111,10 @@ def show view, args
   end
 
   view :closed do |args|
-    frame args.reverse_merge(:content=>true, :body_class=>'closed-content', :toggle_mode=>:close, :optional_toggle=>:show ) do
-      _optional_render :closed_content, args
+    wrap_with :div, :class=>'closed-wrapper' do
+      frame args.reverse_merge(:content=>true, :body_class=>'closed-content', :toggle_mode=>:close, :optional_toggle=>:show ) do
```

Comment:
if the slot the has no `overflow: hidden` then the body adjusts its width to its content